src/main/scala/org/zalando/spearheads/innkeeper/dao/DeletedRoutes.scala

```diff
@@ -0,0 +1,16 @@
+package org.zalando.spearheads.innkeeper.dao
+
+import java.time.LocalDateTime
+
+import org.zalando.spearheads.innkeeper.dao.MyPostgresDriver.api._
+
+object DeletedRoutes extends TableQuery(new DeletedRoutesTable(_))
+
+class DeletedRoutesTable(tag: Tag)
+    extends Table[(String, LocalDateTime)](tag, "DELETED_ROUTES") {
+
+  def name = column[String]("NAME")
+  def deletedAt = column[LocalDateTime]("DELETED_AT")
+
+  def * = (name, deletedAt)
```

Comment:
don't we need other infos like who deleted the route, the initial id of the route, a unique id?